onadata/libs/permissions.py

```diff
@@ -300,47 +347,57 @@ class OwnerRole(Role):
 # Memoize a class to permissions dict.
 for role in ROLES.values():
     role.class_to_permissions = defaultdict(list)
-    [role.class_to_permissions[k].append(p) for p, k in role.permissions]
+    for p, k in role.permissions:
+        role.class_to_permissions[k].append(p)
 
 
 def is_organization(obj):
-    """Some OrganizationProfiles have a pointer to the UserProfile, but no
+    """
+    Some OrganizationProfiles have a pointer to the UserProfile, but no
     UserProfiles do. Check for that first since it avoids a database hit.
     """
     try:
-        hasattr(obj, 'userprofile_ptr') or obj.organizationprofile
-        return True
+        return (hasattr(obj, 'userprofile_ptr') or
+                obj.organizationprofile is not None)
     except OrganizationProfile.DoesNotExist:
         return False
 
 
 def get_role(permissions, obj):
-    for role in reversed(ROLES_ORDERED):
-        if role.has_role(permissions, obj):
-            return role.name
+    """
+    Return the user role for the given obj permissions.
+    """
+    for _role in reversed(ROLES_ORDERED):
+        if _role.has_role(permissions, obj):
+            return _role.name
+    return None
 
 
 def get_role_in_org(user, organization):
+    """
+    Return the user role in the organization.
+    """
     perms = get_perms(user, organization)
 
     if 'is_org_owner' in perms:
         return OwnerRole.name
-    else:
-        return get_role(perms, organization) or MemberRole.name
+
+    return get_role(perms, organization) or MemberRole.name
 
 
 def _get_group_users_with_perms(obj, attach_perms=False, user_perms=None):
     """
     Returns a list of users in the groups with permissions on the object obj.
     """
     if isinstance(obj, XForm):
-        Klass = XFormGroupObjectPermission  # pylint: disable=C0103
```

Comment:
I like the old version better, less repetition, but using a lowercase `klass`

Reply:
I suppose I hear the screams of pylint loader. I will revisit.